.github/ISSUE_TEMPLATE/data-ingest.md

```diff
@@ -19,9 +19,15 @@ Other metadata (_For example: provider, observation_type, model_):
 
 Timeline (_When does this issue need to be completed_):
 
+## Data quality certification
+
+Data ingest requests will not be processed until the following statement is checked. 
+
+- [ ] I certify that this data set has been checked for errors (including malformed or missing data) and tested using Skylab across the entire ingest date range. 
```

Comment:
Is that realistic? Some of the data ingests will be so large that it will take weeks or months to run experiments on all of them?

Reply:
Probably not. What would you use?

Reply:
how about just cross out "across the entire ingest date range" ? That leaves it ambiguous but probably good enough

Reply:
I mean, remove

Reply:
Done.